### PR TITLE
PYTHON-3407 macos release failing on Python 3.8 AttributeError: 'Distribution' object has no attribute 'convert_2to3_doctests'

### PR DESCRIPTION
**Pull request by juliusgeo**

### CODE REVIEW

File: .evergreen/build-mac.sh [hunk 1]
```diff
@@ -12,14 +12,10 @@ for VERSION in 3.7 3.8 3.9 3.10; do
...
-    # Install wheel if not already there.
-    if ! $PYTHON -m wheel version; then
-        createvirtualenv $PYTHON releasevenv
-        WHEELPYTHON=python
-        python -m pip install --upgrade wheel
-    else
-        WHEELPYTHON=$PYTHON
-    fi
+    createvirtualenv $PYTHON releasevenv
+    WHEELPYTHON=python
```
Comment (blink1073) [inline .evergreen/build-mac.sh:16]: Looks like this variable isn't needed anymore and `python setup.py bdist_wheel` can be used below.

**Review by blink1073 — APPROVED** [review]
LGTM!

**Comment by blink1073** [comment]
Don't forget to schedule the release task.

File: .evergreen/build-mac.sh [hunk 2]
```diff
@@ -12,16 +12,10 @@ for VERSION in 3.7 3.8 3.9 3.10; do
...
-        WHEELPYTHON=python
-        python -m pip install --upgrade wheel
-    else
-        WHEELPYTHON=$PYTHON
-    fi
-
-    $WHEELPYTHON setup.py bdist_wheel
+    createvirtualenv $PYTHON releasevenv
+    python -m pip install --upgrade wheel
+    python -m pip install --force-reinstall setuptools==63.2.0
```
Comment (ShaneHarvey) [inline .evergreen/build-mac.sh:17]: Why did you choose 63.2.0?

**Review by ShaneHarvey — APPROVED** [review]

**Pull request merged**